src/styles/index.scss

```diff
@@ -1,3 +1,16 @@
-body {
+@import './blocks/card';
+@import './blocks/stars';
+@import './blocks/variables';
```

Comment:
Consider importing './blocks/variables' before './blocks/card' and './blocks/stars' to ensure that all variables are defined before they are used in these files. This helps prevent any potential issues with undefined variables.